tests/test_async_with.py

```diff
@@ -146,14 +146,15 @@ async def test_create_pool_deprecations(mysql_params, loop):
 @pytest.mark.run_loop
 async def test_sa_connection(table, mysql_params, loop):
     async with sa.create_engine(loop=loop, **mysql_params) as engine:
-        connection = await engine.acquire()
-        assert not connection.closed
-        async with connection:
-            ret = []
-            async for i in connection.execute(tbl.select()):
-                ret.append(i)
+        conn = await engine.acquire()
+        assert not conn.closed
+        async with conn:
+            async with conn.execute(tbl.select()) as cursor:
+                ret = []
+                async for i in cursor:
+                    ret.append(i)
             assert [(1, 'a'), (2, 'b'), (3, 'c')] == ret
-        assert connection.closed
+        assert conn.closed
```

Comment:
I don't think that making variable names worse is required for the code to run under py3.8

Reply:
I have reverted the naming changes. Would be good to choose either `conn` or `connection` and stick to it across all tests, since now it's a mixture of both.

Reply:
Well, consistency is a good idea but it would need to be a separate PR because unrelated stuff makes the review unpleasant experience.